app/views/layouts/ats/_navbar.html.slim

```diff
@@ -1,7 +1,11 @@
-ruby:
-  is_ats_profile = request.fullpath.eql?("/ats/profile")
+- account_icon_dropdown_elements = capture do
+    li.nav-item class=("active" if request.fullpath.eql?("/ats/settings"))
+      = link_to t("core.settings"), ats_settings_path, class: "nav-link"
+    li.nav-item
+      = button_to t("rodauth.logout_button"), rodauth.logout_path, method: :post, class: "nav-link",
+                  form: { data: { turbo: false } }
 
-nav.navbar.navbar-expand-lg.align-items-center.fixed-top.header.gap-2
+nav.navbar.navbar-expand-lg.align-items-center.fixed-top.header.gap-3
```

Comment:
According to the mockup here, you only need a horizontal gap
```suggestion
nav.navbar.navbar-expand-lg.align-items-center.fixed-top.header.column-gap-3
```
`gap-3`
![image](https://github.com/user-attachments/assets/f9054027-c464-4547-8775-2449b994121b)

`column-gap-3`
![image](https://github.com/user-attachments/assets/a75b945b-78c6-4d92-b2de-ff5bdd79a406)